src/service/container-service.ts

```diff
@@ -66,194 +80,43 @@ export interface UpdatedContainerParameters {
    * Filter based on container owner.
    */
   ownerId?: number;
-  returnProducts?: boolean;
-  productId?: number;
-}
-
-/**
- * Define container filtering parameters used to filter query results.
- */
-export interface ContainerParameters extends UpdatedContainerParameters {
-  /**
-   * Filter based on pointOfSale id.
-   */
-  posId?: number;
   /**
-   * Filter based on pointOfSale revision.
+   * Filter on containers related to the given user.
    */
-  posRevision?: number;
-  /**
-   * Whether to select public containers.
-   */
-  public?: boolean;
+  userId?: number;
+  returnProducts?: boolean;
+  productId?: number;
 }
 
 export default class ContainerService {
-  /**
-   * Helper function for the base mapping the raw getMany response container.
-   * @param rawContainer - the raw response to parse.
-   */
-  private static asContainerResponse(rawContainer: any): ContainerResponse {
-    return {
-      id: rawContainer.container_id,
-      revision: rawContainer.container_revision,
-      name: rawContainer.container_name,
-      createdAt: rawContainer.container_createdAt instanceof Date ? rawContainer.container_createdAt.toISOString() : rawContainer.container_createdAt,
-      updatedAt: rawContainer.container_updatedAt instanceof Date ? rawContainer.container_updatedAt.toISOString() : rawContainer.container_updatedAt,
-      public: !!rawContainer.container_public,
+
+  public static revisionToResponse(revision: ContainerRevision): ContainerResponse | ContainerWithProductsResponse {
+    const response: any = {
```

Comment:
Why is the response here `any` and not the correct response type? `ContainerWithProductsResponse` extends `ContainerResponse`.